database/migrations/settings/2024_05_08_104300_create_customer_experience_support_links_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+
+class CreateCustomerExperienceSupportLinksTable extends Migration
+{
+
+    public function up(): void
+    {
+        Schema::create('customer_experience_support_links', function(Blueprint $table){
+            $table->increments('id');
+            $table->string('link_copy')->nullable();
+            $table->string('link_url')->nullable();
```

Comment:
Same here as the previous migration.

```suggestion
            $table->string('copy')->nullable();
            $table->string('url')->nullable();
```